src/components/screens/ScoreSetEditor.vue

```diff
@@ -846,6 +848,22 @@ export default {
           this.licenseId = this.defaultLicenseId
         }
       }
+    },
+    geneName: {
+      handler: async function(newValue, oldValue) {
+        if (newValue == oldValue) {
+          return;
+        }
+        this.geneNameAccessionSuggestions = await this.fetchTargetAccessionsByGene(this.geneName.name.trim())
```

Comment:
this will error if this.geneName is null (which it may be if the dropdown has no value selected) why not the this.geneNameDropdownValue = this.geneName?.name.trim() and then use that here and below?

Reply:
Yes, you're correct. It will throw error if I click the Clear Target button.